### PR TITLE
fizz buzz challenge done

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/assets/README/FizzBuzzTree_README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#FizzBuzzTree
+- Take in a tree and return the same tree as a fizz buzz tree
+
```
Comment (evanslaton) [inline challenges/assets/README/FizzBuzzTree_README.md:3]: What is the Big O efficiency of the method?

File: challenges/src/test/java/Java/FizzBuzzTree/FizzBuzzTreeTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class FizzBuzzTreeTest {
+
+    public BinaryTree<Integer> testing;
+
+
```
Comment (evanslaton) [inline challenges/src/test/java/Java/FizzBuzzTree/FizzBuzzTreeTest.java:17]: Lot's of empty lines!
